joi-validations/users.validations.js

```diff
@@ -1,4 +1,5 @@
-const {object, string, boolean, number} = require('./general.validations');
+const Joi = require('joi');
+const {object, string, boolean, number, uuid} = require('./general.validations');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `uuid` from `./general.validations` is a new addition. Please ensure that the `uuid` validation function exists and works as expected in the `general.validations` module.
```diff
- const {object, string, boolean, number} = require('./general.validations');
+ const {object, string, boolean, number, uuid} = require('./general.validations');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->